app/src/main/kotlin/com/wire/android/util/ImageUtil.kt

```diff
@@ -0,0 +1,76 @@
+package com.wire.android.util
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import android.graphics.Matrix
+import android.net.Uri
+import androidx.exifinterface.media.ExifInterface
+import java.io.ByteArrayOutputStream
+
+/**
+ * Rotates the image to a [ExifInterface.ORIENTATION_NORMAL] in case it's rotated with a different orientation
+ *
+ * @param exif Exif interface for of the image to rotate
+ * @return Bitmap the rotated bitmap or the same in case there is no rotation performed
+ */
+private fun Bitmap.rotateImageToNormalOrientation(exif: ExifInterface?): Bitmap {
+    val orientation = exif?.getAttributeInt(ExifInterface.TAG_ORIENTATION, ExifInterface.ORIENTATION_NORMAL)
+    val matrix = Matrix()
+    when (orientation) {
+        ExifInterface.ORIENTATION_NORMAL -> return this
```

Comment:
do we need to do this : 

 ExifInterface.ORIENTATION_NORMAL -> return this
 
if after everything we do "else" on the value and return this any way ? or has it to do with a nullable oreiantion value ? maybe more clear will be a null check if(orienation == null) and otherwise use a when ? so that is it more implicit what is going on here